### DISCUSSION TITLE
Change validation message on file uploads?

### DISCUSSION THREAD

**Discussion by sinnbeck** [filamentphp/filament#9048] [Help]
I am currently using the `acceptedFileTypes()` method to ensure only pdf and word documents can be uploaded (job application site).

Now the problem is that the mimes for these are
```php
->acceptedFileTypes([
    'application/pdf',
    'application/msword',
    'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
])
```
This will result in a very bad validation message for the end user, which isnt much help at all.
![image](https://github.com/filamentphp/filament/assets/13980140/9bdbba9c-a5be-4105-ae03-994233711867)

Is there an easy way of changing this validation message for this one field? I found nothing in the docs.

I tried adding a messages method as livewire uses, but it does not seem to be called at all
```
public function messages()
    {
        return [
            'data.files.mimetypes' => 'File must be pdf or word',
        ];
    }
```
Is this due to it using filepond which never actually makes a request to the backend?

**Comment by chauhanvishal29** [comment]
Face same issue still nothing get relate to this issue, i want to customise that validation message.
